src/redecomp/transfer/MatrixTransfer.cpp

```diff
@@ -400,11 +414,17 @@ MPIArray<int> MatrixTransfer::buildRecvTrialElemDofs( const RedecompMesh& trial_
               elem_id = trial_p2r_elems.first[src_trial_r][elem_offset];
               auto n_elem_dofs = parent_trial_fes_.GetFE( elem_id )->GetDof() * parent_trial_fes_.GetVDim();
               trial_dofs.resize( trial_dofs.size() + n_elem_dofs );
-              auto dof_array = mfem::Array<int>( &trial_dofs[trial_dofs.size() - n_elem_dofs], n_elem_dofs );
-              parent_trial_fes_.GetElementVDofs( elem_id, dof_array );
-              for ( auto& dof : dof_array ) {
-                dof += first_dof;
+              auto dof_array = mfem::Array<HYPRE_BigInt>( &trial_dofs[trial_dofs.size() - n_elem_dofs], n_elem_dofs );
+              auto int_dof_array = mfem::Array<int>( n_elem_dofs );
```

Comment:
This is in the weeds, but maybe a comment that mfem won't return a BigInt array, and you have to do this workaround?